doc/release_notes.rst

```diff
@@ -5,6 +5,153 @@ Release notes
+    - `npmx ldsw soft_start enable set`
+    - `npmx led mode get`
+    - `npmx led mode set`
+    - `npmx led state set`
+    - `npmx pof polarity get`
+    - `npmx pof polarity set`
+    - `npmx pof status get`
+    - `npmx pof status set`
+    - `npmx pof threshold get`
+    - `npmx pof threshold set`
+    - `npmx ship config inv_polarity get`
+    - `npmx ship config inv_polarity set`
+    - `npmx ship config time get`
+    - `npmx ship config time set`
+    - `npmx ship mode hibernate`
+    - `npmx ship mode ship`
+    - `npmx ship reset long_press get`
+    - `npmx ship reset long_press set`
+    - `npmx ship reset two_buttons get`
+    - `npmx ship reset two_buttons set`
+    - `npmx timer config compare get`
+    - `npmx timer config compare set`
+    - `npmx timer config mode get`
+    - `npmx timer config mode set`
+    - `npmx timer config prescaler get`
+    - `npmx timer config prescaler set`
+    - `npmx timer config strobe`
+    - `npmx timer disable`
+    - `npmx timer enable`
+    - `npmx timer watchdog kick`
+    - `npmx vbusin current_limit get`
+    - `npmx vbusin current_limit set`
+    - `npmx vbusin status cc get`
+
+- Added `CONFIG_NPMX_RESTORE_VALUES` KConfig option that allows for restoring values from PMIC during npmx initialization.
+
+Changed
+~~~~~~~
+
+- Updated the npmx version to 1.0.0.
+- Updated the nrfxlib version to 2.5.0.
+- Aligned samples and shell to the following changes:
+
+    - `npmx_adc_ntc_set()` change to `npmx_adc_ntc_config_set()`
+    - Battery temperature calculation moved to the ADC driver
+    - `NPM1300` define replaced `NPM1300_ENG_C`
+    - `CONFIG_NPMX_DEVICE_NPM1300` Kconfig replaced `CONFIG_NPMX_DEVICE_NPM1300_ENG_C`
+    - Device tree binding `nordic,npmx-npm1300` replaced `nordic,npm1300-eng-c`
+    - Other minor npmx API changes - see npmx v1.0.0 changelog for details
+    - `npmx_core_init()` API
+    - Calling `npmx_timer_task_trigger()` with `NPMX_TIMER_TASK_STROBE` is no longer needed after `npmx_timer_config_set()`
+
+- Interrupt pins (`host-int-gpios` and `pmic-int-pin`) are now optional in devicetree.
+- Split `buck` sample into `buck_output_simple`, `buck_output_advanced`, `buck_pins` and `buck_retention`.
+- Reduced sleep time for voltage stabilization in LDO example from 1 second to 100 ms.
+- Renamed shell commands:
+
+    - `npmx adc meas take vbat` to `npmx adc meas vbat`
+    - `npmx buck set` to `npmx buck status set`
+    - `npmx buck status power get` to `npmx buck status get`
+    - `npmx buck vout {get, set}` to `npmx buck vout_select {get, set}`
+    - `npmx charger charger_current {get, set}` to `npmx charget charging_current {get, set}`
+    - `npmx charger module ntc {get, set}` to `npmx charger module ntc_limits {get, set}`
+    - `npmx charger status get` to `npmx charger status all get`
+    - `npmx charger trickle {get, set}` to `npmx charger trickle_voltage {get, set}`. It now accepts integer value instead of enumeration
```

Comment:
```suggestion
    - `npmx charger trickle {get, set}` to `npmx charger trickle_voltage {get, set}`. It now accepts integer values instead of enumeration.
```